backend/api/views.py

```diff
@@ -237,6 +238,12 @@ class PhotoViewSet(ModelViewSet):
     queryset = Photo.objects.all()
     serializer_class = PhotoSerializer
 
+    def get_permissions(self):
+        if self.action in ['postPhotos', 'getPhotos']:
```

Comment:
Нам только чтение фоток без авторизации нужно. Загрузка фоток, всё равно, пусть требует авторизацию.
 ```suggestion
        if self.action == 'getPhotos':
```